package.json

```diff
@@ -108,6 +109,7 @@
     "redux": "4.0.4",
     "redux-logger": "3.0.6",
     "redux-thunk": "2.2.0",
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
We use it for watching when window is resizing.
First we use ResizeObserver, but Firefox only support it recently in v69. So we use resize-observer-polyfill, it support for all Firefox and chrome.

Reply:
Why are you using `ResizeObserver` instead of  [window.resize](https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event) event? The difference between them is that `window.resize` event fires when window size is changing and `ResizeObserver` can monitor individual elements. Do you need to monitor individual elements?

Reply:
Yes, that right. we only want to observer timeline control. Because when we resize the window height, it's not affect width of timeline bar, so we don't need to reload it.